### PR TITLE
chore(windows): remove old dcpcrypt components 🎢

### PR DESCRIPTION
**Pull request by mcdurdin**
Removes as many references to DCPCrypt as possible.

We have kept DCPRC4 as it is currently used for GlobalProxySettings. Once we eventually remove GlobalProxySettings and depend on Windows config instead, we can remove DCPCrypt altogether.

Note that DCPCrypt should no longer be used for any new code.

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas [hunk 1]
```diff
@@ -635,8 +635,8 @@ procedure TProjectFile.SaveState(node: IXMLNode);   // I4698
 procedure TProjectFile.UpdateID;
 begin
   if Assigned(FProject)
-    then FID := CleanID(MD5Hash(LowerCase( ExtractRelativePath(FProject.FileName, FFileName) )))   // I4709
-    else FID := CleanID(MD5Hash(LowerCase(FFileName)));
+    then FID := CleanID(THashMD5.GetHashString(LowerCase( ExtractRelativePath(FProject.FileName, FFileName) )))   // I4709
+    else FID := CleanID(THashMD5.GetHashString(LowerCase(FFileName)));
```
Comment (mcdurdin) [inline developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas:639]: I have tested and can confirm that `THashMD5.GetHashString` is a drop-in replacement for `MD5Hash`.

Comment (jahorton) [reply developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas:639]: So... what's our dependency for `THashMD5`, then?

File: windows/src/ext/dcpcrypt.txt [hunk 2]
```diff
@@ -3,25 +3,27 @@ Keyman Third Party Package Readme: DcpCrypt
...
-Download from site above.  
+Include Paths:     src\ext\dcpcrypt
 
 Build notes
 -----------
 
-Build package from Developer.  Primary package name is DCPdelphi2009.dpk
+********************************************************************************
+WARNING: THIS PACKAGE IS DEPRECATED. DO NOT DEPEND ON THIS PACKAGE FOR NEW CODE.
+********************************************************************************
```
Comment (jahorton) [inline windows/src/ext/dcpcrypt.txt:18]: :+1:

**Review by jahorton — APPROVED** [review]

File: developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas (hunk 1 above)
Comment (mcdurdin) [reply developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas:639]: `System.Hash` unit -- included with Delphi.

**Pull request merged**